sdk/core/azure-core/src/http/transport_policy.cpp

```diff
@@ -10,7 +13,10 @@ std::unique_ptr<RawResponse> TransportPolicy::Send(
     Request& request,
     NextHttpPolicy nextHttpPolicy) const
 {
-  AZURE_UNREFERENCED_PARAMETER(nextHttpPolicy);
+  if (!nextHttpPolicy.LastPolicy())
```

Comment:
This change doesn't seem relevant to the goal to "remove azure hpp".
Can we split this off into its own PR please?

Reply:
+1

Reply:
it is somehow related. After removing AZURE_UNREF_PARAM(...) , we need to void the parameter.
But that's fine, let's have it apart :)